sdk/go/common/apitype/core.go

```diff
@@ -318,8 +318,10 @@ type ResourceV3 struct {
 	PendingReplacement bool `json:"pendingReplacement,omitempty" yaml:"pendingReplacement,omitempty"`
 	// AdditionalSecretOutputs is a list of outputs that were explicitly marked as secret when the resource was created.
 	AdditionalSecretOutputs []resource.PropertyKey `json:"additionalSecretOutputs,omitempty" yaml:"additionalSecretOutputs,omitempty"`
-	// Aliases is a list of previous URNs that this resource may have had in previous deployments.
-	Aliases []resource.URN `json:"aliases,omitempty" yaml:"aliases,omitempty"`
+	// AliasURNs is a list of previous URNs that this resource may have had in previous deployments.
```

Comment:
I'd ask that we _do not_ change this. It's a headache for dealing with compatibility between different cli versions acting on the same stack.

Reply:
Got it - I'll rename the new field, and revert the old

Reply:
Updated!

Reply:
So I wouldn't even add the new field, like leave ResourceV3 alone completely. There's no need to save the new alias structs here.

Reply:
Got it - I'd love a rundown of how some of these types are used in different places sometime.  It's not entirely obvious to me.